src/main/java/uk/gov/hmcts/reform/civil/service/robotics/mapper/EventHistoryMapper.java

```diff
@@ -691,8 +692,12 @@ private void buildRespondentDivergentResponse(EventHistory.EventHistoryBuilder b
             respondent2ResponseDate = caseData.getRespondent2ResponseDate();
         }
         if (goingOffline) {
-            LocalDateTime queryDate = respondent1ResponseDate.isAfter(respondent2ResponseDate)
-                        ? respondent1ResponseDate : respondent2ResponseDate;
+            //Divergent response can occur in 2v1 where respondent2ResponseDate is not available.
```

Comment:
Found a null pointer with RPA while testing my email notification refactor changes so had to fix this here.